tests/test_examples.py

```diff
@@ -73,9 +73,20 @@ def test_example(self, example_name):
             environ={
                 'ZIPLINE_ROOT': self.tmpdir.getpath('example_data/root'),
             },
-            benchmark_returns=self.benchmark_returns,
+            benchmark_returns=benchmark_returns,
         )
-        expected_perf = self.expected_perf[example_name]
+        if benchmark_returns is not None:
+            expected_perf = self.expected_perf[example_name]
+        else:
+            # With no benchmark, expect zero results for these metrics:
```

Comment:
should we precompute this in the init like `self.no_benchmark_expected_perf`?

Reply:
Good idea. Done.